hw/ip_templates/ac_range_check/data/ac_range_check.tpldesc.hjson

```diff
@@ -9,6 +9,12 @@
       type: "string"
       default: ""
     }
+    {
+      name:    "uniquified_modules"
+      desc:    "A dictionary mapping template_names to uniquified_names"
```

Comment:
This description is a bit vague. Should it say something about what actually gets replaced in templates? (i.e. that it's used implicitly in `instance_vlnv()`)

Probably, each IP ought to describe which modules it could depend on, but we don't have to make that part of this PR.

Reply:
If this is true of all ipgen templates, then we could probably put that info in some common ipgen doc / README or something :thinking: